pkg/scheduler/actions/backfill/backfill.go

```diff
@@ -133,10 +134,35 @@ func backFill(ssn *framework.Session, job *api.JobInfo) {
 			}
 
 			if task.InitResreq.LessEqual(node.Idle) {
+
+				/* patch the annotation */
+				annotation := map[string]string{v1alpha1.BackfillAnnotationKey: "true"}
```

Comment:
I think we need to check the job status here. If job is not ready, we probably should not patch the pod.

We patch and bind pod only if the job is ready.